cmake/h2inc.cmake

```diff
@@ -1,5 +1,5 @@
 function(make_h2inc TARGET SRC DST)
-    set(mkdrv "${CMAKE_SOURCE_DIR}/contrib/mkdrv.pl")
+    set(mkdrv "${BRender_SOURCE_DIR}/contrib/mkdrv.pl")
```

Comment:
Hmm, will that work when compiling directly?

Could you this instead?
```cmake
    set(mkdrv "${CMAKE_CURRENT_FUNCTION_LIST_DIR}/../contrib/mkdrv.pl")
    ```

Reply:
This worked for me when building directly and when building as an external project using FetchContent. `CMAKE_CURRENT_FUNCTION_LIST_DIR` does seem more intuitive though, so I will change it.